lib/src/internet_check_option.dart

```diff
@@ -27,12 +29,17 @@ class InternetCheckOption {
   /// final options = InternetCheckOption(
   ///   uri: Uri.parse('https://example.com'),
   ///   timeout: Duration(seconds: 5),
+  ///   headers: {
+  ///      'Authorization': 'Bearer token',
+  ///   },
   /// );
   /// ```
   InternetCheckOption({
     required this.uri,
     this.timeout = const Duration(seconds: 3),
-  });
+    Map<String, String>? headers,
+    this.responseStatusFn,
+  }) : headers = headers ?? <String, String>{};
```

Comment:
Why initialize headers like this? We can just use:

```dart
this.headers = const {},
```

Reply:
sure, will modify it